internal/bminventory/inventory_test.go

```diff
@@ -13048,3 +13048,90 @@ func validateNetworkConfiguration(cluster *models.Cluster, clusterNetworks *[]*m
 		}
 	}
 }
+
+var _ = Describe("Update cluster - feature usage flags", func() {
+	var (
```

Comment:
The place to test usage is in the subsystem tests. There are several tests there already